src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/SupplierContact.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[SupplierContact.sql]

```

Comment:
Dont need the [] round dbo. SupplierContact should just be SupplierContact, which again doesnt need [] round